apps/api/src/billing/controllers/stripe/stripe.controller.ts

```diff
@@ -150,4 +150,13 @@ export class StripeController {
     const response = await this.stripe.getCustomerTransactions(currentUser.stripeCustomerId, options);
     return { data: response };
   }
+
+  @Protected([{ action: "read", subject: "StripePayment" }])
+  async exportTransactionsCsvStream(options: { startDate: string; endDate: string; timezone: string }): Promise<AsyncIterable<string>> {
+    const { currentUser } = this.authService;
+
+    assert(currentUser.stripeCustomerId, 500, "Payment account not properly configured. Please contact support.");
```

Comment:
```suggestion
    assert(currentUser.stripeCustomerId, 403, "Payments are not configured. Please start with a trial first");
```

Reply:
This assertion was taken from `this.getCustomerTransactions` above, which is now replaced with gracefully returning empty data there. I'm wondering if something similar should be done here or if assertion is better suited still?

```
    if (!currentUser.stripeCustomerId) {
        return 'No transactions found';
    }
```